INN/umbrella-currentorg
# 122: Search functionality

## Changes

This pull request makes the following changes:

- Functioning search on the page
- Functioning LMP
- (hopefully) final names for partials files related to the Projects project
- adds organization name to list items
- adds project status terms and project category terms to list items

## Why



For #122, covering the search functionality.

So that we can have a solid staging branch from which to begin work on #125.

## Testing/Questions

Features that this PR affects:

- The `[current-ltw-projects]` post type


Questions that need to be answered before merging:

- [ ] Does the search work on both cases?
- [x] Does the Load More Posts button work? Are the results it returns results that match the search query?

Steps to test this PR:

1. Put a `[current-ltw-projects]` shortcode on a page
2. On both the `/local-that-works/` archive and on the be-shortcoded page, and verify that the search results change in response to the search. 
3. Click the Load More Posts button, if present
